internal/faustjs.org/docs/gutenberg/tutorial/create-a-block-from-wordpress-core.mdx

```diff
@@ -4,61 +4,66 @@ title: Creating a Block from the WordPress Blocks list
 description: This tutorial will give you some simple steps to follow to create a Block from the official WordPress Blocks list
 ---
 
-In this tutorial you will install and create a block using `@faustwp/blocks` that is using data from the official Blocks that are available in the block editor of your WordPress site.
-It assumes you have already installed both the `@faustwp/blocks` and `wp-graphql-content-blocks` plugin.
+In this tutorial you will install and create a block using `@faustwp/blocks`. It will use data from the official Blocks that are available in the block editor of your WordPress site.
 
-## 1. Set up a `@faustwp/blocks`
-If you have not already done so, you will need to follow the [Getting Started](/docs/gutenberg/getting-started) guide so that you are ready to create new blocks.
+## Prerequisites
 
-## 2. Pick a block that you want to implement
-The Core Editor Blocks contain a list of useful blocks for editing and publishing content.
-All of the official blocks include a `block.json` with defined attributes, so the `wp-graphql-content-blocks` plugin will automatically expose those as GraphQL fields.
+You will need to complete both of these guides before you can create blocks with this tutorial. Make sure you have the latest version of `Faust` and `@faustwp/blocks` installed if you have already completed these guides in the past.
 
-For this tutorial, we are going to implement the [Code Block](https://wordpress.com/support/wordpress-editor/blocks/code-block/) that renders a formatted block of code:
+1. [Faust Getting Started Guide](/docs/getting-started)
+2. [Blocks Getting Started Guide](/docs/gutenberg/getting-started)
 
-## 3. Review the Block design in the Editor
+## 1. Choose a block to implement
 
-Before you try to implement the block, it's useful to get an idea of how the block is configured within the editor.
-You want to create a few sample blocks and test their options.
+:::note
+The Core Editor Blocks contain a list of useful blocks for editing and publishing content. All of the official blocks include a `block.json` with defined attributes, so the `wp-graphql-content-blocks` plugin will automatically expose those as GraphQL fields.
+:::
+
+For this tutorial, we are going to implement the [Code Block](https://wordpress.com/support/wordpress-editor/blocks/code-block/) that renders a formatted block of code.
```

Comment:
Docs should only reference wordpress.org.

- wordpress.org (free & open-source)
- wordpress.com (commercial)

```suggestion
For this tutorial, we are going to implement the [Code Block](https://developer.wordpress.org/block-editor/reference-guides/core-blocks/#code) that renders a formatted block of code.
```